sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -55,15 +55,21 @@ object CosmosAccountConfig {
     parseFromStringFunction = accountEndpointUri => accountEndpointUri,
     helpMessage = "Cosmos DB Account Key")
 
+  val CosmosConsistency = CosmosConfigEntry[String](key = "spark.cosmos.accountConsistency",
```

Comment:
```suggestion
  val CosmosConsistency = CosmosConfigEntry[ConsistencyLevel](key = "spark.cosmos.accountConsistency",
```

Reply:
No longer relevant, this config doesn't exist anymore